docs: improve readme

AntoineRR opened the pull request:
**Description**

This is a proposal for a new readme with updated information.
Also the readme was  duplicated in the `docs` folder. I kept only the one in the `docs` folder as it should appear on the repo homepage too as stated in [this](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes) documentation. This will make us maintain only one readme instead of trying to keep both in sync.

sansyrox commented on docs/README.md:
```diff
@@ -3,16 +3,17 @@
-Robyn is an async Python backend server with a runtime written in Rust, btw.
+Robyn is a fast async Python web framework coupled with a web server written in Rust.
+
+Check out the talk at **PyCon Sweden 2021** about [Robyn: An async python web framework with a Rust runtime](https://www.youtube.com/watch?v=DK9teAs72Do)
```
I think we should have a separate talk section. But we can address this later. 😄

sansyrox replied:
Or, rather , we can just link it here - https://github.com/sansyrox/robyn/blob/main/docs/community-resources.md

AntoineRR replied:
That's a good point, updated it

sansyrox reviewed (commented):
All looks great! But, would this not replace the README preview on the main GitHub preview?

AntoineRR commented:
The `docs/README.md` file should indeed appear as a preview on the repo main page.
I've been thinking about this though, and wonder if we shouldn't have a completely different README.md for the docs (that will appear in the home page of the docs)? Not sure it makes sense to have the readme both in the repo and on the docs home page

sansyrox commented:
> I've been thinking about this though, and wonder if we shouldn't have a completely different README.md for the docs (that will appear in the home page of the docs)? 

We can do that but what will be different in both of them?

AntoineRR commented:
I updated the `docs/README.md` with a more documentation centered approach, LMK if that suits you.
Also added the Robyn logo as the docs website icon!

sansyrox commented on docs/README.md:
````diff
@@ -1,101 +1,20 @@
-from robyn import Robyn
-
-app = Robyn(__file__)
-
-
-@app.get("/")
-async def h(request):
-    return "Hello, world!"
-
-app.start(port=8080)
-```
-
-## 💡 Features
-
-- Under active development!
-- Written in Rust, btw xD
-- A multithreaded Runtime
-- Extensible
-- A simple API
-- Sync and Async Function Support
-- Dynamic URL Routing
-- Multi Core Scaling
-- WebSockets!
-- Middlewares
-- Hot Reloading
-- Community First and truly FOSS!
-
-## 🗒️ Contributor Guidelines
-
-Feel free to open an issue for any clarification or for any suggestions.
-
-If you're feeling curious. You can take a look at a more detailed architecture [here](https://github.com/sansyrox/robyn/blob/main/docs/architecture.md).
-
-## ⚙️ To Develop Locally
-
-1. Install the development dependencies: `pip install -r dev-requirements.txt`
-
-1. Install the pre-commit git hooks: `pre-commit install`
-
-1. Add more routes in the `integration_tests/base_routes.py` file(if you like).
````
@AntoineRR , I think we should not delete this. If people will want to have a quick start documentation summary, they will have to go to the previous page. But if we keep it here, they won't have to move away from the documentation.

I believe that we need a proper custom documentation website revamp but I think we can keep this for now.

What say? @AntoineRR

AntoineRR replied:
Ok that makes sense, let's keep this readme as a copy of the one on the repo for now then. We'll see later how to deal with this properly.

sansyrox commented on docs/index.html:
```diff
@@ -12,6 +12,7 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0, minimum-scale=1.0"
     />
+    <link rel="icon" href="https://user-images.githubusercontent.com/29942790/140995889-5d91dcff-3aa7-4cfb-8a90-2cddf1337dca.png" />
```
Niceee

sansyrox reviewed (approved):
Great work @AntoineRR . LGTM! 👍
